htslib/hts_defs.h

```diff
@@ -111,4 +111,12 @@ DEALINGS IN THE SOFTWARE.  */
 #define HTSLIB_EXPORT
 #endif
 
+#if defined(_WIN32) || defined(__MSYS__)
+#define HTS_PATH_SEPARATOR_CHAR ';'
+#define HTS_PATH_SEPARATOR_STR  ";"
+#else
+#define HTS_PATH_SEPARATOR_CHAR ':'
+#define HTS_PATH_SEPARATOR_STR  ":"
+#endif
+
```

Comment:
This could be private to HTSlib, alongside the `hts_path_itr` stuff in _hts_internal.h_.

Or if it's decided to make it public, it would be better in _hts_os.h_ or perhaps _hts.h_ — but not really _hts_defs.h_ which is about compiler capabilities.

Reply:
The “other tricks” BTW is the following (though if made public it's probably best to provide both of them anyway rather than making users needing a string use this trick too);

```c
static const char path_sep_string[] = { HTS_PATH_SEPARATOR_CHAR, '\0' };
```